lib/timers.js

```diff
@@ -345,6 +347,7 @@ Timeout.prototype.close = function() {
   } else {
     exports.unenroll(this);
   }
+  return this;
 };
```

Comment:
Node's timer API is pretty different from what browsers use. Browsers return numerical IDs while we return Timeout objects. I'd say if there's no public use case, we should move it to `_close`. In the long run, I'd like to see us getting closer to the standardized [WindowTimers](http://www.w3.org/TR/html5/webappapis.html#windowtimers) which means returning numerical values, and possibly intruduce helper methods for ref/unref functionality.